src/provider/provider_tracking.c

```diff
@@ -585,13 +597,20 @@ static umf_result_t trackingOpenIpcHandle(void *provider, void *providerIpcData,
 
     ret = umfMemoryProviderOpenIPCHandle(p->hUpstream, providerIpcData, ptr);
     if (ret != UMF_RESULT_SUCCESS) {
+        LOG_ERR(
+            "tracking open ipc handle: umfMemoryProviderOpenIPCHandle failed");
```

Comment:
I see that you start other log messages with a capital letter when its not a function name.

Reply:
I refactored code: I changed LOG_* macros and added __func__ to it to automatically add function name to the log message.